src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCompiler.cs

```diff
@@ -99,7 +100,7 @@ internal abstract class RegexCompiler
         protected string[]? _strings;                                              // the stringtable associated with the RegexCodes
         protected (string CharClass, bool CaseInsensitive)[]? _leadingCharClasses; // the possible first chars computed by RegexPrefixAnalyzer
         protected RegexBoyerMoore? _boyerMoorePrefix;                              // a prefix as a boyer-moore machine
-        protected int _anchors;                                                    // the set of anchors
+        protected int _leadingAnchor;                                                    // the set of anchors
```

Comment:
(nit) is the whitespace on the comment off?

Reply:
Looks like it.  Will fix.